patches/chrome-browser-ui-android-toolbar-java-src-org-chromium-chrome-browser-toolbar-top-ToolbarTablet.java.patch

```diff
@@ -1,8 +1,16 @@
 diff --git a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
-index e2751bf67b045c558010ea0d2f57e2c3b7db2077..53023c144a5df7af2a727cf6dfa1b44dfd5d43d2 100644
+index e2751bf67b045c558010ea0d2f57e2c3b7db2077..d34673607845e18f4d779a9fc65010784f2710c3 100644
 --- a/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
 +++ b/chrome/browser/ui/android/toolbar/java/src/org/chromium/chrome/browser/toolbar/top/ToolbarTablet.java
-@@ -306,6 +306,7 @@ public class ToolbarTablet extends ToolbarLayout implements OnClickListener {
+@@ -195,6 +195,7 @@ public class ToolbarTablet extends ToolbarLayout implements OnClickListener {
+             forward();
+             RecordUserAction.record("MobileToolbarForward");
+         }
++        BraveToolbarLayout.class.cast(this).onClickImpl(v);
```

Comment:
shouldn't we subclass `ToolbarTablet`?

Reply:
we definitely need to start creating unit tests for these overrides so this kind of thing doesn't happen

Reply:
This one in particular is just a revert of the change that wasn't meant to be done during rebase.
Tests would be great, but we don't run them on tablets so this issue probably wouldn't be caught. Robolectric tests would help, but we don't have them running in Brave at the moment.
As for subclassing `ToolbarTablet` and `ToolbarPhone`, we can do it, but then we will have to patch (or likely copy over) corresponding `xml` files, which is also not great, so we have to choose between 2 not great options.

Reply:
I think we can do this pretty cleanly if we change the name of the original class to `ToolbarLayoutChromiumImpl` and then rename our subclass to to `ToolbarLayout` so it properly inflates as our class instead of theirs?

Reply:
I'm pretty sure we have some robolectric tests, but if not we need to start adding them because we need more tests on android.